main.c

```diff
@@ -5,6 +5,14 @@ struct node {
         int item;
     };
 
+void printList(struct node* head) {
+  while (head != NULL) {
+    printf("%d -> ", head->item);
+    head = head->nxt;
+  }
+  printf("NULL\n");
+}
+

```

Comment:
I added this because your `printNode` function was producing kinda fugly output ah, sorry.